driver-core/src/main/com/mongodb/ReadConcern.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright 2015 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.mongodb;
+
+import org.bson.BsonDocument;
+import org.bson.BsonString;
+
+import static com.mongodb.assertions.Assertions.notNull;
+
+/**
+ * The readConcern option allows clients to choose a level of isolation for their reads.
+ *
+ * The WiredTiger storage engine, introduces the readConcern option for replica sets and replica set shards.  Allowing clients to choose a
+ * level of isolation for their reads.
```

Comment:
Couple of comments about this Javadoc:

1.  Though the 3.2 release notes mention WiredTiger specifically, I don't think the Javadocs should, as other storage engines may also support read concern.
2.  Neither of the sentences are complete grammatical sentences.  ;)